src/topology/support.lean

```diff
@@ -0,0 +1,178 @@
+/-
+Copyright (c) 2022 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn, Patrick Massot
+-/
+
+import topology.separation
+
+/-!
+# The topological support of a function
+
+In this file we define the topological support of a function `f`, `tsupport f`,
+as the closure of the support of `f`.
+
+Furthermore, we say that `f` has compact support if the topological support of `f` is compact.
+
+## Main definitions
+
+* `function.mul_support`
+* `function.has_compact_mul_support`
```

Comment:
could you also mention the additive names here? Also, mul_support should be mul_tsupport on the first line.